README.rst

```diff
@@ -15,11 +15,56 @@ simphony-kratos is hosted on github: https://github.com/simphony/simphony-kratos
 Requirements
 ------------
 
-- somphony-common > 0.1.1
+- `simphony-common`_ == 0.1.3
+
+.. _simphony-common: https://github.com/simphony/simphony-common
 
 Installation
 ------------
 
+Installation is based on setuptools and requeires Python 2.7::
+
+    # build and install
+    python setup.py install
+
+or::
+
+    # build for in-place development
+    python setup.py develop
+
 Kratos Installation
--------------------
+~~~~~~~~~~~~~~~~~~~
+
+Both Kratos-CFD and DEMPack wrappers use KratosMultiphysics. A suitable version to run with these wrappers can be downloaded here:
+
+- `KratosMultiphysics`_
+
+.. _KratosMultiphysics: https://web.cimne.upc.edu/users/croig/data/kratos-simphony.tgz
+
```

Comment:
Maybe additonal information could be added about downloadable version of Kratos:

This version is prepared for Ubuntu-12-04-LTS 64-bit.  See http://kratos-wiki.cimne.upc.edu/index.php/Download for additional information on getting Kratos for a specific system.